js/signin.js

```diff
@@ -0,0 +1,100 @@
+
+const userEmail = document.querySelector("#userEmail")
+const errorMessage = document.querySelectorAll(".errorMessage")
```

Comment:
배열임을 나타내기 위해 복수형으로 작성해주시면 좋을 것 같아요 :)

```suggestion
const errorMessages = document.querySelectorAll(".errorMessage")
```

Reply:
이름이 더 길어지더라도 더 명시적인 이름을 가질 수 있도록 각각이 무엇을 의미하는지 이름을 지어두는 것이 더 좋을 것 같아요. errorMessages[0], errorMessages[1] 과 같이 순서에 의존해서 인덱스로 접근하는 것은 가독성을 떨어뜨릴 수 있어요.

```js
const emailErrorMessage = errorMessages[0]
const passwordErrorMessage = errorMessages[1]
```